pkgs/servers/code-server/default.nix

```diff
@@ -21,28 +21,29 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "code-server";
-  version = "4.0.1";
-  commit = "7fe23daf009e5234eaa54a1ea5ff26df384c47ac";
+  version = "4.8.1";
+  commit = "ca182b9fb51e2b1683d6e154ba5086fc7e8c3238";
 
   src = fetchFromGitHub {
     owner = "cdr";
     repo = "code-server";
     rev = "v${version}";
-    sha256 = "1s3dcmzlkyh7qfs3ai1p7dlp45iys0ax1fbxxz17p395pw9anrrl";
+    fetchSubmodules = true;
```

Comment:
Do we really need to do this? submodules are not known to be super reproducible

Reply:
upstream has changed to use submodules since previous derivation
build process requires that the submodule is pulled specifically in the directory specified so I believe it is required
if there is a way to pull this code via nix and then put it in the required location that is worth investigating but would increase maintenance complexity

Reply:
using vscode as a submodule from upstream is alright.